back/spec/services/participation_context_service_spec.rb

```diff
@@ -4,14 +4,60 @@
   let(:service) { ParticipationContextService.new }
 
   describe 'participation_possible_for_context?' do
-    it 'returns true when participation is possible' do
-      project = create(:continuous_project, posting_enabled: true)
-      expect(service.participation_possible_for_context?(project, create(:user))).to be_truthy
+    let(:no_participation_attributes) do
+      {
+        participation_method: 'ideation',
+        posting_enabled: false,
+        voting_enabled: false,
+        commenting_enabled: false
+      }
+    end
+
+    it 'returns true when posting is possible' do
+      expect(service).to receive(:posting_idea_disabled_reason_for_context).and_return nil
+      project = create :continuous_project, no_participation_attributes
+      expect(service.participation_possible_for_context?(project, create(:user))).to eq true
```

Comment:
Please write `.to be true` instead of `.to eq true`, also elsewhere in this file. RuboCop will be happy :-)